torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -100,6 +100,7 @@ std::unordered_map<IterDomain*, IterDomain*> PairwiseRootDomainMap::map(
   }
 
   std::unordered_map<IterDomain*, IterDomain*> dom_map;
+  
```

Comment:
Why do we touch this line?

Reply:
This change makes no sense. Some previous modification was not deleted.